src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -1248,6 +1248,10 @@
       <Project>{6e0a079e-85f1-45a1-ad5b-9855e4344809}</Project>
       <Name>Units</Name>
     </ProjectReference>
+    <ProjectReference Include="..\Libraries\PythonNodeModels\PythonNodeModels.csproj">
+      <Project>{8872CA17-C10D-43B9-8393-5C5A57065EB0}</Project>
+      <Name>PythonNodeModels</Name>
```

Comment:
hmm @zeusongit so now DynamoCoreWPF depends on PythonNodeModels, please add `Private False` to this entry to see if it helps the failing test issue.